plugins/mumble_plugin.h

```diff
@@ -118,7 +82,6 @@ typedef struct _MumblePluginQt {
 } MumblePluginQt;
 
 typedef MumblePlugin *(MUMBLE_PLUGIN_CALLING_CONVENTION *mumblePluginFunc)();
-typedef MumblePlugin2 *(MUMBLE_PLUGIN_CALLING_CONVENTION *mumblePlugin2Func)();
 typedef MumblePluginQt *(MUMBLE_PLUGIN_CALLING_CONVENTION *mumblePluginQtFunc)();
 
 /*
```

Comment:
Outdated documentation below

Reply:
To be removed.

Reply:
Uh. Let's update it instead? ;) I'm fine with that happening once this PR is technically sound but it is important documentation.

Reply:
I meant that we should remove the outdated documentation and then rewrite it from scratch, sorry.